### ISSUE TITLE
evdevremapkeys does not work on suspend

### ISSUE THREAD

**Issue by unkissedfrog** [philipl/evdevremapkeys#11]
Hi, 
evdevremapkeys works great, but there's ohne issue: it stops working after waking up from sleep. Any idea how to fix that?

Thanks

**Comment by gliviu** [comment]
Resuming from standby works fine in my case. Maybe some device ids change. You could use `sudo python3 evdevremapkeys.py --list-devices` before entering standby. Run the command again after resuming and see the differences.

**Comment by philipl** [comment]
Yeah, you should be able to find a way to identify your device that doesn't change.

**Comment by unkissedfrog** [comment]
I think, I solved it: created a systemd service for evdevremapkeys and another service that is run on resume. The resume service just waits 2 seconds, before restarting the evdrevremapkeys service... perhups not the best solution, but for me it works. I assume, that weather my mouse or the entire USB-System is not quick enough in waking up. The device IDs remain unchanged, so I do not know the exact reason. But I found a way to work around and now I can full enjoy this great peace of software. Many thanks for developing it!
